phpBB/includes/acp/acp_extensions.php

```diff
@@ -1006,6 +1051,7 @@ public function list_available_exts(\phpbb\extension\manager $phpbb_extension_ma
 	*/
 	private function output_actions($block, $actions)
 	{
+		$vars_ary = [];
```

Comment:
if using short array syntax, maybe use short array syntax in the entire function for consistency.